_queries.ts

```diff
@@ -15,16 +15,20 @@ export interface Query<T> {
 
 const fetcher = async (
   url: string,
-  fetchToken: getAccessTokenSilently,
+  fetchToken: getAccessTokenSilently | null = null,
```

Comment:
Setting the default parameter value already makes it optional, and since the method parameter is also optional with a default value, I don't think the order of these two parameters matter. I might be missing something, or perhaps you overlooked the default values?